testing/go/pgcatalog_test.go

```diff
@@ -598,7 +598,7 @@ func TestPgType(t *testing.T) {
 				},
 				{ // Different cases but non-quoted, so it works
 					Query:    "SELECT typname FROM PG_catalog.pg_TYPE ORDER BY typname;",
-					Expected: []sql.Row{},
+					Expected: []sql.Row{{"anyarray"}, {"bool"}, {"bpchar"}, {"bytea"}, {"char"}, {"date"}, {"float4"}, {"float8"}, {"int2"}, {"int4"}, {"int8"}, {"json"}, {"jsonb"}, {"name"}, {"numeric"}, {"oid"}, {"text"}, {"time"}, {"timestamp"}, {"timestamptz"}, {"timetz"}, {"unknown"}, {"uuid"}, {"varChar"}, {"xid"}},
```

Comment:
For this test, we should filter to a few known ones, otherwise we'll be updating this test whenever we add a new type.